app/views/playerControls.js

```diff
@@ -215,6 +223,13 @@ const Player = Mn.ItemView.extend({
         this.render();
     },
 
+    expand(e) {
+        e.preventDefault();
+        let player = document.querySelector('.player')
```

Comment:
Always same concerns: prefer to trigger an `player:expand`event and react to it in the player's view